<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <title>PATh</title>
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <link rel="icon" type="image/png" href="/images/logo/Logo_Round_32x32.png">
    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display:wght@400;700&display=swap" rel="stylesheet">
    <link href="/assets/css/style.css" rel="stylesheet">
    
    <meta property="og:title" content="Partnership to Advance Throughput Computing" />
    <meta property="og:type" content="website" />
    <meta property="og:url" content="https://path-cc.io" />
    <meta property="og:image" content="https://path-cc.io/images/logo/Logo_Primary_Acronym.png" />
    <meta property="og:description" content="PATh brings together the Center for High Throughput Computing and the Open Science Grid in order to advance the nation’s campuses and science communities through the use of distributed High Throughput Computing." />
    <meta name="twitter:card" content="summary" />
    <meta name="twitter:site" content="PATh" />
    <meta name="twitter:creator" content="" />
    <meta name="twitter:image" content="images/logo/Logo_Round_Med.png" />

    <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.9.2/dist/umd/popper.min.js" integrity="sha384-IQsoLXl5PILFhosVNubq5LC7Qb9DXgDA9i+tQ8Zj3iwWAwPtgFTxbJ8NT4GN1R8p" crossorigin="anonymous"></script>
    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.min.js" integrity="sha384-cVKIPhGWiC2Al4u+LWgxfKTRIcfu0JTxR+EQDz/bgldoEyl4H0zUF0QKbrJ0EcQF" crossorigin="anonymous"></script>
</head>
<body class='page page-services-list'>
<div id="wrapper" class="wrapper">
    


<div>
  <div class="container-lg">
    <nav class="navbar navbar-expand-lg navbar-light ps-0 pe-0">

      <a class="navbar-brand" href="/">
        <img height="70px" src="/images/logo/Logo_Round_Med.png" alt="PATh"/>
      </a>
      <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNavDropdown" aria-controls="navbarNavDropdown" aria-expanded="false" aria-label="Toggle navigation">
        <span class="navbar-toggler-icon"></span>
      </button>
      <div id="navbarNavDropdown" class="collapse navbar-collapse justify-content-between">
        <ul class="navbar-nav">
          <li class="nav-item dropdown">
            <a class="nav-link dropdown-toggle" href="#" id="about-nav" role="button" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
              About
            </a>
            <div class="dropdown-menu" aria-labelledby="about-nav">
              <a class="dropdown-item" href="/about">PATh</a>
              <a class="dropdown-item" href="/team">Team</a>
              <a class="dropdown-item" href="/glossary">Glossary</a>
            </div>
          </li>
          <li class="nav-item dropdown">
            <a class="nav-link dropdown-toggle" href="#" id="facility-nav" role="button" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
              Facility
            </a>
            <div class="dropdown-menu" aria-labelledby="facility-nav">
              <a class="dropdown-item" href="/facility/index.html">About</a>
              <a class="dropdown-item" href="/facility/registration.html">Registration</a>
            </div>
          </li>
          
          
          <li class="nav-item ">
            <a class="nav-link" href="/services/">Services</a>
          </li>
          
          <li class="nav-item active">
            <a class="nav-link" href="/news/">News</a>
          </li>
          
          <li class="nav-item ">
            <a class="nav-link" href="/events/">Events</a>
          </li>
          
          <li class="nav-item ">
            <a class="nav-link" href="/contact/">Contact</a>
          </li>
          
        </ul>
        <div class="social-media d-none d-lg-block">
          <a class="text-decoration-none" href="https://twitter.com/opensciencegrid">
            <img alt="Twitter Icon" src="/images/icons/twitter.svg"/>
          </a>
          <a class="text-decoration-none" href="https://www.youtube.com/channel/UCd1UBXmZIgB4p85t2tu-gLw">
            <img alt="Youtube Icon" src="/images/icons/youtube.svg"/>
          </a>
        </div>
      </div>
    </nav>
  </div>
</div>

    <main>
        <div class="container pb-5">
            <div class="pt-5">
    <h1 class="title mb-3"></h1>
<h1 class="mb-0 d-flex flex-row justify-content-between">
    
    
</h1>


</div>
<h1>News Articles</h1>

<div class="row">
  




  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2022-11-03-ucsd-external-release/">PATh Extends Access to Diverse Set of High Throughout Computing Research Programs</a>
          
          <small class="text-muted article-date">November 03, 2022</small>
          
        </h2>
        <p>UCSD announces the new PATh Facility and discusses its impact on science.
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2022-09-27-DoIt-Article-Summary/">Solving for the future: Investment, new coalition levels up research computing infrastructure at UW–Madison</a>
          
          <small class="text-muted article-date">September 27, 2022</small>
          
        </h2>
        <p>Summary of Corissa Runde’s article from the UW-Madison Department of Information Technology website.
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2022-09-09-NIAID-ACE-students-attend-OSG-User-School/">NIAID/ACE students attend this year’s OSG User School 2022</a>
          
          <small class="text-muted article-date">September 09, 2022</small>
          
        </h2>
        <p>This past July, the OSG User School 2022 welcomed students from across the globe to learn how to use high-throughput computing (HTC) in their scientific research.

This year five students from Makerere University in Uganda and the University Of Sciences, Techniques, and Technologies of Bamako in Mali, Africa, participated as a part of The U.S. National Institute of Allergy and Infectious Diseases (NIAID) and the African Centers for Excellence in Bioinformatics and Data-Intensive Science (ACE) partnership program.
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2022-07-29-OSG-User-School-Concludes/">OSG User School Concludes</a>
          
          <small class="text-muted article-date">July 29, 2022</small>
          
        </h2>
        <p>After a week of participating in the OSG User School 60+ students are being released to use impact research across the globe. In this one week event, students learn to run large-scale computing workloads at their campus or using the national-scale OSPool provided by the OSG Consortium.
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2022-07-21-Messick/">LIGO's Search for Gravitational Waves Signals Using HTCondor</a>
          
          <small class="text-muted article-date">July 21, 2022</small>
          
        </h2>
        <p>Cody Messick, a Postdoc at the Massachusetts Institute of Technology (MIT) working for the LIGO lab, describes LIGO’s use of HTCondor to search for new gravitational wave sources.
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2022-07-21-EOL-OSG/">Retirements and New Beginnings: The Transition to Tokens</a>
          
          <small class="text-muted article-date">July 21, 2022</small>
          
        </h2>
        <p>May 1, 2022 officially marked the retirement of OSG 3.5, GridFTP, and GSI dependencies. OSG 3.6, up and running since February of 2021, is prepared for usage and took its place, relying on WebDAV and bearer tokens.
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2022-07-12-Wilcots/">The Future of Radio Astronomy Using High Throughput Computing</a>
          
          <small class="text-muted article-date">July 12, 2022</small>
          
        </h2>
        <p>Eric Wilcots, UW-Madison dean of the College of Letters &amp; Science and the Mary C. Jacoby Professor of Astronomy, dazzles the HTCondor Week 2022 audience.
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2022-07-06-Opotowsky/">Expediting Nuclear Forensics and Security Using High Throughput Computing</a>
          
          <small class="text-muted article-date">July 06, 2022</small>
          
        </h2>
        <p>Arrielle C. Opotowsky, a 2021 Ph.D. graduate from the University of Wisconsin-Madison’s Department of Engineering Physics, describes how she utilized high throughput computing to expedite nuclear forensics investigations.
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2022-07-06-Hiemstra/">Testing GPU/ML Framework Compatibility</a>
          
          <small class="text-muted article-date">July 06, 2022</small>
          
        </h2>
        <p>Justin Hiemstra, a Machine Learning Application Specialist for CHTC’s GPU Lab, discusses the testing suite developed to test CHTC’s support for GPU and ML framework compatibility.
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2022-07-06-Gillett/">UW Statistics Course using HTC</a>
          
          <small class="text-muted article-date">July 06, 2022</small>
          
        </h2>
        <p>For the first time, UW Statistics undergraduates could participate in a course teaching high throughput computing (HTC). John Gillett, lecturer of Statistics at the University of Wisconsin-Madison, designed and taught the course with the support of the Center for High Throughput Computing (CHTC).
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2022-07-06-Garcia/">Using HTC and HPC Applications to Track the Dispersal of Spruce Budworm Moths</a>
          
          <small class="text-muted article-date">July 06, 2022</small>
          
        </h2>
        <p>Matthew Garcia, a Postdoctoral Research Associate in the Department of Forest &amp; Wildlife Ecology at the University of Wisconsin–Madison, discusses how he used the HTCondor Software Suite to combine HTC and HPC capacity to perform simulations that modeled the dispersal of budworm moths.
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2022-07-06-Fulvio/">Using high throughput computing to investigate the role of neural oscillations in visual working memory</a>
          
          <small class="text-muted article-date">July 06, 2022</small>
          
        </h2>
        <p>Jacqueline M. Fulvio, lab manager and research scientist for the Postle Lab at the University of Wisconsin-Madison, explains how she used the HTCondor Software Suite to investigate neural oscillations in visual working memory.
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2022-06-02-Record/">OSPool Hits Record Number of Jobs</a>
          
          <small class="text-muted article-date">June 02, 2022</small>
          
        </h2>
        <p>The OSPool processed over 2.6 million jobs during the week of April 14th - 17th this year and ran over half a million jobs on two separate days that week.
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2022-06-01-PATh-Facility/">Introducing the PATh Facility: A Unique Distributed High Throughput Computing Service</a>
          
          <small class="text-muted article-date">June 01, 2022</small>
          
        </h2>
        <p>Researchers can now request credits on the PATh Facility, the PATh project’s new service intended for distributed high throughput computing workflows supporting NSF science.
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2022-06-01-Johri/">The role of HTC in advancing population genetics research</a>
          
          <small class="text-muted article-date">June 01, 2022</small>
          
        </h2>
        <p>Postdoctoral researcher Parul Johri uses OSG services, the HTCondor Software Suite, and the population genetics simulation program SLiM to investigate historical patterns of genetic variation.
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2022-06-01-HTCondorWeek-Photos/">A Long-Awaited Reunion: HTCondor Week 2022 in Photos</a>
          
          <small class="text-muted article-date">June 01, 2022</small>
          
        </h2>
        <p>HTCondor Week 2022 featured over 40 exciting talks, tutorials, and research spotlights focused on the HTCondor Software Suite (HTCSS). Sixty-three attendees reunited in Madison, Wisconsin for the long-awaited in-person meeting, and 111 followed the action virtually on Zoom.
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2022-05-27-HTCondor-Week-Concludes/">HTCondor Week 2022 Concludes</a>
          
          <small class="text-muted article-date">May 27, 2022</small>
          
        </h2>
        <p>Thank you to all in-person and virtual participants in HTCondor Week 2022. Over the course
of the event we had over 40 talks spanning tutorials, applications and science domains using HTCSS.

We hope to see you next year!
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2022-05-12-EHT/">High-throughput computing as an enabler of black hole science</a>
          
          <small class="text-muted article-date">May 12, 2022</small>
          
        </h2>
        <p>The stunning new image of a supermassive black hole in the center of the Milky Way was created by eight telescopes, 300 international astronomers and more than 5 million computational tasks. This Morgridge Institute article describes how the Wisconsin-based Open Science Pool helped make sense of it all.
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2022-05-02-NIAID/">NIAID/ACE - OSG collaboration leads to a successful virtual training session</a>
          
          <small class="text-muted article-date">May 02, 2022</small>
          
        </h2>
        <p>The U.S. National Institute of Allergy and Infectious Diseases (NIAID) and the African Centers for Excellence in Bioinformatics and Data-Intensive Science (ACE) partnered with the OSG Consortium to host a virtual high throughput computing training session for graduate students from Makerere University and the University Of Sciences, Techniques, and Technologies of Bamako (USTTB).
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2022-05-02-CLAS12/">Expanding, uniting, and enhancing CLAS12 computing with OSG’s fabric of services</a>
          
          <small class="text-muted article-date">May 02, 2022</small>
          
        </h2>
        <p>A mutually beneficial partnership between Jefferson Lab and the OSG Consortium at both the organizational and individual levels has delivered a prolific impact for the CLAS12 Experiment.
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2022-04-25-Natzke/">Learning and adapting with OSG: Investigating the strong nuclear force</a>
          
          <small class="text-muted article-date">April 25, 2022</small>
          
        </h2>
        <p>David Swanson Memorial Award winner, Connor Natzke’s journey with the OSG Consortium began in 2019
as a student of the OSG User School.
Today, nearly three years later, Natzke has executed 600,000 simulations with the help of OSG staff
and prior OSG programming. These simulations, each of them submitted as a job, logged over 135,000
core hours provided by the Open Science Pool (OSPool). Natzke’s history with the OSG Consortium
reflects a pattern of learning, adapting, and improving that translates to the acceleration and
expansion of scientific discovery.
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2021-03-19-Nicholas-Cooley-2021-David-Swanson-Award-Winner/">Connor Natzke awarded David Swanson Memorial Award</a>
          
          <small class="text-muted article-date">April 11, 2022</small>
          
        </h2>
        <p>Connor Natzke was awarded the 2022 David Swanson Memorial Award at the March OSG All-Hands Meeting. The memorial was established to honor our late colleage David Swanson who contributed to campus research across the country.
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2022-04-11-OSGAHM2022-Summary/">Celebrating a dynamic OSG All-Hands Meeting 2022</a>
          
          <small class="text-muted article-date">April 11, 2022</small>
          
        </h2>
        <p>In March, 251 OSG users, resource providers, and staff convened virtually for the OSG All-Hands Meeting 2022. This article provides a brief summary of the talks and discussions that took place, and includes links to the video recordings of all talks.
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2022-02-22-ML-and-Image-Analyses-for-Livestock-Data/">Machine Learning and Image Analyses for Livestock Data</a>
          
          <small class="text-muted article-date">February 22, 2022</small>
          
        </h2>
        <p>In this presentation from HTCondor Week 2021, Joao Dorea from the Digital Livestock Lab explains how high-throughput computing is used in the field of animal and dairy sciences.
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2021-12-16-precision-mental-health/">Harnessing HTC-enabled precision mental health to capture the complexity of smoking cessation</a>
          
          <small class="text-muted article-date">December 16, 2021</small>
          
        </h2>
        <p>Collaborating with CHTC research computing facilitation staff, UW-Madison researcher Gaylen Fronk is using HTC to improve cigarette cessation treatments by accounting for the complex differences among patients.
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2021-11-09-usgs-invasivespecies/">Protecting ecosystems with HTC</a>
          
          <small class="text-muted article-date">November 09, 2021</small>
          
        </h2>
        <p>Researchers at the USGS are using HTC to pinpoint potential invasive species for the United States.
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2021-10-26-bat-genomics/">80,000 jobs, 40 billion base pairs, and 20 bats –– all in 4 weeks</a>
          
          <small class="text-muted article-date">October 26, 2021</small>
          
        </h2>
        <p>An evolutionary biologist at the AMNH used HTC services provided by the OSG to unlock a genomic basis for convergent evolution in bats.
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2021-10-26-Banq/">Centuries of newspapers are now easily searchable thanks to HTCSS</a>
          
          <small class="text-muted article-date">October 26, 2021</small>
          
        </h2>
        <p>BAnQ’s digital collections team recently used HTCSS to tackle their largest computational endeavor yet –– completing text recognition on all newspapers in their digital archives.
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2021-09-23-Resilience/">Resilience: How COVID-19 challenged the scientific world</a>
          
          <small class="text-muted article-date">September 23, 2021</small>
          
        </h2>
        <p>In the face of the pandemic, scientists needed to adapt. This article by the Morgridge Institute for Research provides a thoughtful look into how individuals and organizations, including the CHTC, have pivoted in these challenging times.
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2021-08-19-Spencer-Showcase/">Scaling virtual screening to ultra-large virtual chemical libraries</a>
          
          <small class="text-muted article-date">August 19, 2021</small>
          
        </h2>
        <p>Kicking off the OSG User School Showcase, Spencer Ericksen, a researcher at the University of Wisconsin-Madison’s Carbone Cancer Center, described how high throughput computing (HTC) has made his work in early-stage drug discovery infinitely more scalable.
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2021-08-19-Showcase/">Transforming research with high throughput computing</a>
          
          <small class="text-muted article-date">August 19, 2021</small>
          
        </h2>
        <p>During the OSG Virtual School Showcase, three different researchers shared how high throughput computing has made lasting impacts on their work.
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2021-08-19-Hannah-Showcase/">Using HTC for a simulation study on cross-validation for model evaluation in psychological science</a>
          
          <small class="text-muted article-date">August 19, 2021</small>
          
        </h2>
        <p>During the OSG School Showcase, Hannah Moshontz, a postdoctoral fellow at UW-Madison’s Department of Psychology, described her experience of using high throughput computing (HTC) for the very first time, when taking on an entirely new project within the field of psychology.
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2021-08-19-Anirvan-Showcase/">Antimatter: Using HTC to study very rare processes</a>
          
          <small class="text-muted article-date">August 19, 2021</small>
          
        </h2>
        <p>Anirvan Shukla, a User School participant in 2016, spoke at this year’s Showcase about how high throughput computing has transformed his research of antimatter in the last five years.
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2021-01-15-Campus-Cyberinfrastructure-Awardees-Contribute-to-OSG/">Campus Cyberinfrastructure (CC*) Awardees Contribute Over 349 Million Core Hours in Last 12 Months</a>
          
          <small class="text-muted article-date">August 12, 2021</small>
          
        </h2>
        <p>Campus Cyberinfrastructure Award Recipients Power the OSG and contributed over 349 million core hours to researchers using distributed high throughput computing (dHTC).
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2021-08-10-Science-Gateway/">OSG fuels a student-developed computing platform to advance RNA nanomachines</a>
          
          <small class="text-muted article-date">August 10, 2021</small>
          
        </h2>
        <p>How undergraduates at the University of Nebraska-Lincoln developed a science gateway that enables researchers to build RNA nanomachines for therapeutic, engineering, and basic science applications.
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2021-06-10-OSPool-Hits-Record-Utilization/">OSPool Usage Hits Daily Record</a>
          
          <small class="text-muted article-date">June 10, 2021</small>
          
        </h2>
        <p>Researchers utilizing the OSPool are racking up record-breaking numbers. On June 8, the OSPool, which provides computing resources to researchers across the country, went over 1.1 million core hours –– a daily record number. To put this in perspective, one million core hours is equivalent to using 42 thousand cores in just one day. That is close to half the size of some large supercomputing centers. In short, an increasing number of researchers are utilizing the OSG to carry out an incredible amount of computing.

</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2021-01-15-Using-HTCondor-For-Large-File-Transfer/">How to Transfer 460 Terabytes? A File Transfer Case Study</a>
          
          <small class="text-muted article-date">January 15, 2021</small>
          
        </h2>
        <p>When Greg Daues at the National Center for Supercomputing Applications (NCSA) needed to transfer 460 Terabytes of NCSA files from the National Institute of Nuclear and Particle Physics (IN2P3) in Lyon, France to Urbana, Illinois, for a project they were working with FNAL, CC-IN2P3 and the Rubin Data Production team, he turned to the HTCondor High Throughput system, not to run computationally intensive jobs, as many do, but to manage the hundreds of thousands of I/O bound transfers.

</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2020-12-18-OSG-Training-and-Campus-Workshop/">Register Now for the February 8-9 Campus Workshop</a>
          
          <small class="text-muted article-date">December 18, 2020</small>
          
        </h2>
        <p>Save the date and register now for another Campus Workshop on distributed high-throughput computing (dHTC) February 8-9 offered by the Parternship to Advance Throughput Computing (PATh).
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2020-12-7-Submit-Locally-Run-Globally/">Submit Locally, Run Globally</a>
          
          <small class="text-muted article-date">December 07, 2020</small>
          
        </h2>
        <p>The PATh project offers technologies and services that enable researchers to harness through a single interface, and from the comfort of their “home directory”, computing capacity offered by a global and diverse collection of resources.
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2020-11-18-High-Throughput-Computing-Powers-Covid-Genetic-Sequencing/">High Throughput Computing Powers Covid Genetic Sequencing</a>
          
          <small class="text-muted article-date">November 18, 2020</small>
          
        </h2>
        <p>MADISON - David O’Connor, professor at the UW School of Medicine and Public Health, reports that recent advances in genetic sequencing of viruses makes it possible to track the spread of SARS-CoV-2 throughout communities in Wisconsin.
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2020-10-20-path-awarded/">National Science Foundation establishes a partnership to advance throughput computing </a>
          
          <small class="text-muted article-date">October 10, 2020</small>
          
        </h2>
        <p>MADISON — Recognizing the University of Wisconsin-Madison’s leadership role in research computing, the National Science Foundation announced this month that the Madison campus will be home to a five-year, $22.5 million initiative to advance high-throughput computing.

</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/htc-in-support-of-science/">High Throughput Computing in Support of Science</a>
          
        </h2>
        <p>The high throughput computing capabilities provided by HTCondor and the OSG Consortium’s Fabric of Services have a rich history of advancing all domains of research. From detecting gravitational waves caused by ancient black hole collisions, to hunting viral variants of COVID-19 –– browse the collection of articles below to discover just what’s possible with high throughput computing.

</p>
      </div>
    </div>
  </div>
  
</div>


  
        </div>
    </main>
</div>
<div class="footer">
  <div class="container">
    <div class="row">
      <div class="col-12">
        <div class="footer-inner">
          <h3 class="footer-title">PATh</h3>
          <ul>
            
            
            <li class="">
              <a href="/">Home</a>
            </li>
            
            <li class="">
              <a href="/contact/">Contact</a>
            </li>
            
          </ul>
        </div>
      </div>
    </div>
  </div>
</div>
<div class="sub-footer">
  <div class="container">
    <div class="row">
      <div class="col-12">
        <div class="sub-footer-inner">
          <div class="social">
              <a href="https://www.nsf.gov/oac">
                  <img alt="NSF Logo" height="70px" width="70px" src="/images/logo/nsf-logo.svg">
              </a>
          </div>
          <div class="copyright">
              This project is supported by National Science Foundation under Cooperative Agreement
              <a href="https://www.nsf.gov/awardsearch/showAward?AWD_ID=2030508">OAC-2030508</a>.
              Any opinions, findings, conclusions or recommendations expressed in this material are those of the
              authors and do not necessarily reflect the views of the National Science Foundation.
          </div>
        </div>
      </div>
    </div>
  </div>
</div>

<script type="text/javascript" src="/assets/js/scripts.js"></script>

    
    
    <script async src="https://www.googletagmanager.com/gtag/js?id=G-0CCSDQV212"></script>
    <script>
      window.dataLayer = window.dataLayer || [];

      function gtag() {
        dataLayer.push(arguments);
      }
      gtag('js', new Date());
      gtag('config', 'G-0CCSDQV212');
    </script>
    

</body>
</html>